docs/content/guides/recipes/howto-private-bicep-registry/index.md

```diff
@@ -0,0 +1,73 @@
+---
+type: docs
+title: "How-To: Pull Bicep Recipes from private OCI container registry."
+linkTitle: "Private bicep registries"
+description: "Learn how to setup your Radius environment to use Bicep Recipe templates published to a private OCI container registry."
+weight: 500
+categories: "How-To"
+tags: ["recipes", "bicep"]
+---
+
+This how-to guide will describe how to:
+
+- Configure a Radius environment to utilize Bicep Recipe templates that are stored in a private OCI (Open Container Initiative) complaint container registry. This setup will ensure the templates are securely stored and managed within private OCI registry.
+
+### Prerequisites
+
+Before you get started, you'll need to make sure you have the following tools and resources:
+
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Radius initialized with `rad init`]({{< ref howto-environment >}})
+
+## Step 1: Setting up private OCI container registry authentication.
+Radius supports three authentication methods for accessing private container registries:
+- Basic Authentication: This method uses a username and password for authentication and is applicable to all OCI complaint registries. Obtain the username and password details used to login to private registry.
```

Comment:
are there examples of basic auth (docker, github registries etc.) that you may include here to illustrate what is meant by "basic authentication"?

```suggestion
- Basic Authentication: This method uses a username and password for authentication and is applicable to all OCI complaint registries. Obtain the `username` and `password` details used to login to private registry.
```